### PR TITLE
Migrate vapor pressure data batch2

### PR DESCRIPTION
**Pull request by sumitnegi7**

### CODE REVIEW

**Review by qur786 — APPROVED** [review]
@sumitnegi7 All data is correct

File: src/data/vapor-pressure-constants.ts [hunk 1]
```diff
@@ -18,6 +18,306 @@ export const DIPPRVaporPressureDictionary: Record<
...
+    formula: 'C7H8S',
+    CAS: '100-53-8',
+    C1: 118.02,
+    C2: -10527,
+    C3: -13.91,
+    C4: 6.4794e-6,
+    C5: 2,
+    minimumTemperature: 243.95,
+    maximumTemperature: 718,
+    pressureAtMinimumTemperature: 2.98e1,
```
Comment (maneetgoyal) [inline src/data/vapor-pressure-constants.ts:63]:
```suggestion
    pressureAtMinimumTemperature: 2.98e-1,
```

File: src/data/vapor-pressure-constants.ts [hunk 2]
```diff
@@ -18,6 +18,306 @@ export const DIPPRVaporPressureDictionary: Record<
...
+    formula: 'C4H10',
+    CAS: '106-97-8',
+    C1: 66.343,
+    C2: -4363.2,
+    C3: -7.046,
+    C4: 9.4509e-6,
+    C5: 2,
+    minimumTemperature: 134.86,
+    maximumTemperature: 425.12,
+    pressureAtMinimumTemperature: 6.74e1,
```
Comment (maneetgoyal) [inline src/data/vapor-pressure-constants.ts:183]:
```suggestion
    pressureAtMinimumTemperature: 6.74e-1,
```

File: src/data/vapor-pressure-constants.ts [hunk 3]
```diff
@@ -18,6 +18,306 @@ export const DIPPRVaporPressureDictionary: Record<
...
+    formula: 'C4H10O2',
+    CAS: '584-03-2',
+    C1: 103.28,
+    C2: -11548,
+    C3: -10.925,
+    C4: 4.26e-18,
+    C5: 6,
+    minimumTemperature: 220,
+    maximumTemperature: 680,
+    pressureAtMinimumTemperature: 2.93e4,
```
Comment (maneetgoyal) [inline src/data/vapor-pressure-constants.ts:198]:
```suggestion
    pressureAtMinimumTemperature: 2.93e-4,
```

File: src/data/vapor-pressure-constants.ts [hunk 4]
```diff
@@ -18,6 +18,306 @@ export const DIPPRVaporPressureDictionary: Record<
...
+    formula: 'C4H10O2',
+    CAS: '107-88-0',
+    C1: 123.22,
+    C2: -12620,
+    C3: -13.986,
+    C4: 0.000003926,
+    C5: 2,
+    minimumTemperature: 196.15,
+    maximumTemperature: 676,
+    pressureAtMinimumTemperature: 3.74e7,
```
Comment (maneetgoyal) [inline src/data/vapor-pressure-constants.ts:213]:
```suggestion
    pressureAtMinimumTemperature: 3.74e-7,
```

File: src/data/vapor-pressure-constants.ts [hunk 5]
```diff
@@ -18,6 +18,306 @@ export const DIPPRVaporPressureDictionary: Record<
...
+    formula: 'C4H10O',
+    CAS: '71-36-3',
+    C1: 106.29483,
+    C2: -9866.35511,
+    C3: -11.6553,
+    C4: 1.08e-17,
+    C5: 6,
+    minimumTemperature: 183.85,
+    maximumTemperature: 563.1,
+    pressureAtMinimumTemperature: 2.91e4,
```
Comment (maneetgoyal) [inline src/data/vapor-pressure-constants.ts:228]:
```suggestion
    pressureAtMinimumTemperature: 2.91e-4,
```

File: src/data/vapor-pressure-constants.ts [hunk 6]
```diff
@@ -18,6 +18,306 @@ export const DIPPRVaporPressureDictionary: Record<
...
+    formula: 'C4H10O',
+    CAS: '78-92-2',
+    C1: 122.552,
+    C2: -10236.2,
+    C3: -14.125,
+    C4: 2.36e-17,
+    C5: 6,
+    minimumTemperature: 158.45,
+    maximumTemperature: 535.9,
+    pressureAtMinimumTemperature: 1.24e6,
```
Comment (maneetgoyal) [inline src/data/vapor-pressure-constants.ts:243]:
```suggestion
    pressureAtMinimumTemperature: 1.24e-6,
```

File: src/data/vapor-pressure-constants.ts [hunk 7]
```diff
@@ -18,6 +18,306 @@ export const DIPPRVaporPressureDictionary: Record<
...
+    C4: 0.000011319,
+    C5: 2,
+    minimumTemperature: 167.62,
+    maximumTemperature: 428.6,
+    pressureAtMinimumTemperature: 7.45e1,
+    pressureAtMaximumTemperature: 4.1e6,
+  },
+
+  Butylacetate: {
+    name: 'trans-2-Butene',
```
Comment (maneetgoyal) [inline src/data/vapor-pressure-constants.ts:293]: Wrong name

File: src/data/vapor-pressure-constants.ts [hunk 8]
```diff
@@ -18,6 +18,306 @@ export const DIPPRVaporPressureDictionary: Record<
...
+    C4: 0.00001047,
+    C5: 2,
+    minimumTemperature: 199.65,
+    maximumTemperature: 575.4,
+    pressureAtMinimumTemperature: 8.17e2,
+    pressureAtMaximumTemperature: 3.087e6,
+  },
+
+  Butylbenzene: {
+    name: 'trans-2-Butene',
```
Comment (maneetgoyal) [inline src/data/vapor-pressure-constants.ts:308]: Name

File: src/data/vapor-pressure-constants.ts [hunk 9]
```diff
@@ -18,6 +18,306 @@ export const DIPPRVaporPressureDictionary: Record<
...
+    formula: 'C6H12O2',
+    CAS: '123-86-4',
+    C1: 122.82,
+    C2: -9253.2,
+    C3: -14.99,
+    C4: 0.00001047,
+    C5: 2,
+    minimumTemperature: 199.65,
+    maximumTemperature: 575.4,
+    pressureAtMinimumTemperature: 8.17e2,
```
Comment (maneetgoyal) [inline src/data/vapor-pressure-constants.ts:303]:
```suggestion
    pressureAtMinimumTemperature: 8.17e-2,
```

**Review by maneetgoyal — CHANGES_REQUESTED** [review]
Corrections needed

**Review by maneetgoyal — APPROVED** [review]

**Pull request merged**